qt/python/mantidqt/utils/qt/__init__.py

```diff
@@ -11,15 +11,16 @@
 """
 from __future__ import absolute_import
 
+import os.path as osp
```

Comment:
Any reason for moving this to the top of the list above the comment? It would be nice to keep things ordered as: stdlib modules, 3rdparty modules, local modules.

Reply:
Autoformatting, it gets confused by comments, thanks for pointing it out!